datalake/constructs/batch_job_queue.py

```diff
@@ -2,6 +2,9 @@
 
 from aws_cdk import aws_batch, aws_dynamodb, aws_ec2, aws_iam, core
 
+APPLICATION_NAME_TAG_NAME = "ApplicationName"
```

Comment:
I don't think this file is a good place for setting `APPLICATION_NAME_TAG_NAME` and `APPLICATION_NAME` variables. I would use for example `..endpoints.utils` instead.